recipes-tag/nsls2ptycho/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: nsls2ptycho
+  version: "1.0.0"
+
+source:
+  git_rev: packaging
+  git_url: https://github.com/leofang/ptycho_gui.git
+
+build:
+  number: 0
+  skip: True # [not py36 or win]
+
+requirements:
+  build:
+    - python 
+    - cython
+    - pip
+    - fftw >=3
+    - qt >=5
+    - setuptools
+  run:
+    - python
+    - fftw >=3
+    - pyfftw
+    - mpi4py 
+    - qt >=5
+    - pyqt >=5
+    - numpy
+    - scipy
+    - matplotlib
+    - pillow
+    - h5py
+    - posix_ipc
+
```

Comment:
it is better to have some import test here.

Reply:
test for the dependencies or the package itself?

Reply:
two import tests added